tests/fixture_layouts/routes/async/_layout.tsx

```diff
@@ -1,15 +1,14 @@
-import { LayoutProps, RouteContext } from "$fresh/server.ts";
+import { LayoutContext } from "$fresh/server.ts";
 
 export default async function AsyncLayout(
   req: Request,
-  ctx: RouteContext,
-  { Component }: LayoutProps,
+  ctx: LayoutContext,
 ) {
   await new Promise((r) => setTimeout(r, 10));
```

Comment:
```suggestion
  await delay(10);
```
Nit. Just for consistency.

Reply:
At some point I'll remember that we have this in our standard library 👀